DigitalLearningSolutions.Web/Controllers/TrackingSystem/Delegates/ViewDelegateController.cs

```diff
@@ -68,5 +68,20 @@ public IActionResult SendWelcomeEmail(int delegateId)
 
             return View("WelcomeEmailSent", model);
         }
+
+        [Route("DeactivateDelegate")]
+        public IActionResult DeactivateDelegate(int delegateId)

```

Comment:
Given that we don't have a confirmation page for the delegates, should we make this POST?

Also, should we add a few controller test (no delegate user, wrong centre and default cases)?